### ISSUE TITLE
[Skyblock] spawn location

### ISSUE THREAD

**Issue by ghost** [Blood-Asp/IndustryGigant#12]
At the Skyblock mode the respawn on death is not at 0;x;0 so you'll spawn in the void and insta-die again

**Comment by Blood-Asp** [comment]
Do you use modpack version 0.5.0 with IGCore 0.2.1? Because before that version the described thing would happen, but now it should work fine. The world has to be generated on that version too.

**Comment by ghost** [comment]
Yep, downloaded via Curse. First tried to download at github but BQ-edit mode is enabled by default so i considored the curse version to be the current one (or you missed to commit the change ;-) )
